pytest_fixtures/component/provision_pxe.py

```diff
@@ -229,6 +232,8 @@ def module_ssh_key_file():
 @pytest.fixture
 def provisioning_host(module_ssh_key_file, pxe_loader, module_provisioning_sat):
     """Fixture to check out blank VM"""
+    if pxe_loader.vm_firmware == 'bios':
+        pytest.skip('BIOS is not supported with IPv6')
```

Comment:
Why skipping here explicitely, when we're skipping it in the test?

Reply:
Since we are skipping bios for IPv6, provisioning host being the function scoped fixture I don't want the host to be created unnecessarily for bios as it would run teardown as well which is a waste of execution time for us. I missed the IPv6 condition here which I will update